package.json

```diff
@@ -15,6 +15,7 @@
     "i18n_extract": "BABEL_ENV=i18n fedx-scripts babel src --quiet > /dev/null",
     "is-es5": "es-check es5 ./dist/*.js",
     "lint": "fedx-scripts eslint --ext .js --ext .jsx .",
+    "lint:fix": "fedx-scripts eslint --ext .js --ext .jsx . --fix",
```

Comment:
Nit: this can be achieved through `npm run lint -- --fix`

Reply:
Yup. I'm just trying to reduce the diff with the template application on which this is based: https://github.com/edx/frontend-template-application/blob/41e8fd4043a9434de20ff7ec2df0452051ebefae/package.json#L18